draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx

```diff
@@ -199,1106 +199,175 @@ export const Assertive = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-quick-engagement-survey.svg"
-  />
-)
-
-export const ValuesSurvey1 = ({ enableAspectRatio, ...props }: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-values-survey-1.svg"
-  />
-)
-
-export const ValuesSurvey2 = ({ enableAspectRatio, ...props }: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-values-survey-2.svg"
-  />
-)
-
-export const WellbeingSurvey1 = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-wellbeing-survey-1.svg"
-  />
-)
-
-export const WellbeingSurvey2 = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-wellbeing-survey-2.svg"
-  />
-)
-
-export const WellbeingSurvey3 = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-wellbeing-survey-3.svg"
-  />
-)
-
-export const ChangeReadiness = ({ enableAspectRatio, ...props }: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-change-readiness.svg"
-  />
-)
-
-export const ChangeSuccess = ({ enableAspectRatio, ...props }: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-change-success.svg"
-  />
-)
-
-export const PerformanceDiagnostics = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-performance-diagnostics.svg"
-  />
-)
-
-export const LeadingThroughCrisis = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-leading-through-crisis.svg"
-  />
-)
-
-export const EmergencyResponse = ({
-  enableAspectRatio,
-  ...props
-}: SpotProps) => (
-  <Base
-    aspectRatio={enableAspectRatio ? "square" : undefined}
-    {...props}
-    name="illustrations/heart/spot/template-library-emergency-response.svg"
-  />
-)
+/* prettier-ignore */ export const BenefitsSurvey = createSpotIllustration("illustrations/heart/spot/template-library-benefits-survey.svg")
```

Comment:
Would be nice if we could specify a rule or ignore a range, but as you said, apparently it's not easy to do.

I wonder if prettier-eslint would make things better. But that can be for a future investigation.